src/mod/endpoints/mod_sofia/mod_sofia.h

```diff
@@ -405,6 +405,9 @@ struct mod_sofia_globals {
 	char guess_mask_str[16];
 	int debug_presence;
 	int debug_sla;
+	int limit_uuid_failure_code; // SIP RC For when insufficient cpu-idle
+	char limit_uuid_failure_text[32]; // The text portion of the SIP response
```

Comment:
length 32 is too small, some of our custom messages can reach more than 32 characters. I would suggest between 64-128 character or make it dynamic and use mem pool to allocate it.

Reply:
Roger that, increased to 128.